core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -273,6 +273,7 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/ScrollSyncService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/AnchorHeaderDirective.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarModeService.js"></script>
```

Comment:
Place this below the `SimpleEditorSidebarDirective.js` (ie swap this and the one below), we alphabetize the imports.